docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,259 @@ Postprocessing
+rules as
+:math:`p_0 \cdot \text{operation}_0(\text{score}) + p_1 \cdot \text{operation}_1(\text{score})`.
+The goal in this case with equalized odds is to reach equal true and false
+positive rates, which are conveniently plotted on the y- and x-axes.
+It follows that they'd be equal at any spot where the two curves intersect.
+We can relax this a little bit since in many cases one lies strictly above
+the other, just like in this example.
+Imagine the diagonal of the ROC plot from :math:`(0,0)` to :math:`(1,1)`.
+It is usually not considered as useful since we could achieve the same result
+with random classifiers with probabilities to get label 1 ranging from 0 to 1.
+If we have one curve strictly below the other, we can still get their true and
+false positive rates to match by "pulling" the higher one towards the
+diagonal. In reality we consider the overlap, so only the points on the lower
+curve are available to us. From those points we choose the one with the best
+value in terms of the overall objective (not per group).
+
+Mathematically, the following formula represents this interpretation of the
+chart, where :math:`c` represents the `prediction_constant` from the printed
+dictionary above.
+
+.. math::
+
+    p_{\text{ignore}} \cdot c + (1-p_{\text{ignore}}) \cdot \left(p_0 \cdot \text{operation}_0(\text{score}) + p_1 \cdot \text{operation}_1(\text{score})\right)
+
+In other words, we create a linear combination of the binary thresholding rule
+outputs. :math:`p_0` and :math:`p_1` indicate how close we are to the points
+representing the two thresholding rules, and :math:`p_0+p_1=1`.
+:math:`p_{\text{ignore}}` is :math:`0` if the curve does not need to be "pulled"
+towards the diagonal, i.e., when the selected solution lies on the curve
+itself as opposed to below.
+Above, the curve for "Male" is strictly below the curve for "Female", so
+:math:`p_{\text{ignore}}=0` for "Male" and non-zero for "Female".
+
+Importantly, :math:`p_{\text{ignore}}` is only required for equalized odds.
+Equalized odds is somewhat more complex than demographic parity or others in
+that it requires equality between two metrics, namely true and false positive
+rates.
+We want to stress on the fact that it is crucial to understand the
+implications of such a result.
+Predictions of the resulting `ThresholdOptimizer` model are determined
+randomly based on the probabilities to predict label 1 that the formula
+produces.
+In this particular example :code:`operation1` will always be true since every
+score is larger than negative infinity.
+For "Male" that means
+
+- predict 1 if the score is greater than :math:`0.5`.
+- predict 0 with probability :math:`0.999926` if the score is less or equal to
+  :math:`0.5`. This means there is a very low probability of getting label 1
+  regardless of the features.
+
+For "Female" the result includes the :math:`p_{\text{ignore}}` term, but one of the
+thresholds is set to always be true
+
+- predict 1 with probability
```

Comment:
Should we include the "predict 0" rule for "Female" to make it consistent with the "Male" section?

Reply:
Consistency is worth rewriting this. I've thought about it a bit and the "Female" case is the much more complicated one due to the extra term with `p_ignore`, so I've instead adjusted the "Male" section to be consistent. I hope that makes sense for you.